crates/fluvio-cli/src/connector/mod.rs

```diff
@@ -165,23 +165,35 @@ impl From<ConnectorConfig> for ManagedConnectorSpec {
         if let Some(producer) = config.producer {
             if let Some(linger) = producer.linger {
                 let linger = humantime::format_duration(linger).to_string();
-                parameters.insert("producer-linger".to_string(), linger);
+                parameters.insert(
+                    "producer-linger".to_string(),
+                    ManageConnectorParameterValue::from(vec![linger]),
+                );
             }
             if let Some(compression) = producer.compression {
                 let compression = format!("{:?}", compression);
-                parameters.insert("producer-compression".to_string(), compression);
+                parameters.insert(
+                    "producer-compression".to_string(),
+                    ManageConnectorParameterValue::from(vec![compression]),
```

Comment:
I think that thid don't need the `vec!`
```suggestion
                    ManageConnectorParameterValue::from(compression),
```